third_party/amd/backend/compiler.py

```diff
@@ -36,6 +36,14 @@ def get_gfx9_limits(lhsType, rhsType):
     return lambda lhsType, rhsType: (1, 1, 1) if is_fma_supported(lhsType.scalar, rhsType.scalar) else (16, 16, 16)
 
 
+def is_pingpong_enabled(arch):
+    if arch == 'gfx942':
+        use_block_pingpong = os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG", "1") == "1"
+    else:
+        use_block_pingpong = os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG", "1") == "1"
```

Comment:
These two are the same at the moment? Do you intend to only enable for cdna3?

Reply:
Ugh.. right. stupid mistake.